samza-core/src/main/scala/org/apache/samza/container/SamzaContainer.scala

```diff
@@ -76,6 +76,49 @@ object SamzaContainer extends Logging {
         classOf[JobModel])
   }
 
+  // Ideally, we want this to reside in TaskStorageManager since container should not be in the business of
```

Comment:
If this is not an actionable, lets remove this comment. Otherwise let's create a ticket and make a TODO.